Resources/Locale/en-US/_RMC14/tiles/tiles.ftl

```diff
@@ -28,6 +28,7 @@ tiles-cm-sand = sand tile
 tiles-cm-grass = grass tile
 tiles-cm-dirt-grass = dirt grass tile
 tiles-cm-rock = rock tile
+tiles-cm-red-desert = red desert tile
```

Comment:
`tiles-rmc-red-desert = red desert tile`
Make sure to replace in .YML file as well